connect/runtime/src/main/java/org/apache/kafka/connect/runtime/isolation/DelegatingClassLoader.java

```diff
@@ -353,4 +359,24 @@ private void addAllAliases() {
             }
         }
     }
+
+    private static class InternalReflections extends Reflections {
+
+        public InternalReflections(final Configuration configuration) {
+            super(configuration);
+        }
+
+        // When Reflections is used for parallel scans, it has a bug where it propagates ReflectionsException
+        // as RuntimeException.  Override the scan behavior to emulate the singled-threaded logic.
+        protected void scan(URL url) {
```

Comment:
We definitely need `@Override` annotation to make clear that we change how `scan(url)` will function when is called by `scan()` on the base class.

Reply:
Thanks, I will add it.